public/app/core/components/AppChrome/ExtensionSidebar/ExtensionSidebarProvider.tsx

```diff
@@ -0,0 +1,57 @@
+import { createContext, ReactNode, useContext, useEffect, useState } from 'react';
+
+import { ComponentTypeWithExtensionMeta, store } from '@grafana/data';
+import { usePluginComponents } from '@grafana/runtime';
+
+export const EXTENSION_SIDEBAR_EXTENSION_POINT_ID = 'grafana/extension-sidebar/v0-beta';
+const EXTENSION_SIDEBAR_DOCKED_LOCAL_STORAGE_KEY = 'grafana.navigation.extensionSidebarDocked';
+const ENABLED_EXTENSION_SIDEBAR_PLUGINS = ['grafana-investigations-app'];
```

Comment:
The idea here is to limit this extension point to a few approved plugins. This would likely be investigations and the dash app.

Reply:
How would it work if you have multiple activated? Which one is active?

Reply:
If multiple "sidebar enabled" plugins are installed, the toolbar button would [show a menu](https://github.com/grafana/grafana/pull/102626/files#diff-d7d8dfe321c642cffb6c5b29a91a76bca109e110d16b93f05c974990a4436718R29-R42).

![image](https://github.com/user-attachments/assets/ded14a9c-f62a-4f57-9998-583323e942fd)

The `dockedPluginId` actually controls which one to render or not.